src/status_im/contexts/chat/messenger/placeholder/style.cljs

```diff
@@ -1,16 +1,15 @@
 (ns status-im.contexts.chat.messenger.placeholder.style
   (:require [quo.foundations.colors :as colors]
-            [react-native.reanimated :as reanimated]))
+            [react-native.safe-area :as safe-area]))
 
 (defn container
-  [top opacity z-index theme]
-  (reanimated/apply-animations-to-style
-   {:opacity opacity
-    :z-index z-index}
-   {:position         :absolute
-    :padding-top      top
-    :top              0
-    :left             0
-    :right            0
-    :bottom           0
-    :background-color (colors/theme-colors colors/white colors/neutral-95 theme)}))
+  [theme on-layout-done?]
+  {:position         :absolute
+   :padding-top      (safe-area/get-top)
+   :top              0
+   :left             0
+   :right            0
+   :bottom           0
+   :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
+   :opacity          (if on-layout-done? 0 1)
+   :z-index          (if on-layout-done? 0 2)})
```

Comment:
We had some shared values to control the logic here, but we didn't do any animations, and a `ratom` can do the job, (this is the piece of code I mentioned works simpler with a ratom)